modules/exploits/windows/misc/webdav_delivery.rb

```diff
@@ -0,0 +1,161 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::EXE
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => 'Serve DLL via webdav server',
+      'Description'  => %q(
+        This module simplifies the rundll32.exe Application Whitelisting Bypass technique.
+        The module creates a webdav server that hosts a dll file. When the user types the provided rundll32
+        command on a system, rundll32 will load the dll remotly and execute the provided export function.
+        The export function needs to be valid, but the default meterpreter function can be anything.
+        The process does write the dll to C:\Windows\ServiceProfiles\LocalService\AppData\Local\Temp\TfsStore\Tfs_DAV
+        but does not load the dll from that location. This file should be removed after execution.
+        The extension can be anything you'd like, but you don't have to use one. Two files will be
+        written to disk. One named the requested name and one with a dll extension attached.
+      ),
+      'License'      => MSF_LICENSE,
+      'Author'       =>
+        [
+          'Ryan Hanson <ryan.hanson@optiv.com>', # research  discovery (@ryhanson)
+          'James Cook <james.cook@optiv.com>',  # MSF Module (@_jbcook)
```

Comment:
No more comma after the last element in a array.
And the format of email should be like `James Cook <james.cook[at]optiv.com>`